cmd/ethstore-exporter/main.go

```diff
@@ -56,6 +60,27 @@ func main() {
 	defer db.ReaderDb.Close()
 	defer db.WriterDb.Close()
 
-	exporter.StartEthStoreExporter(*bnAddress, *enAddress, *updateInterval, *errorInterval, *sleepInterval)
+	var startDayReexport int64 = -1
+	var endDayReexport int64 = -1
+
+	if *daysToReexport != "" {
+		s := strings.Split(*daysToReexport, "-")
+		if len(s) < 2 {
+			logrus.Fatalf("invalid arg")
```

Comment:
Please use `utils.LogFatal` and consider adding some more information like what is expected.